ui/pages/home/home.component.js

```diff
@@ -58,10 +59,6 @@ import FlaskHomeFooter from './flask/flask-home-footer.component';
 
 const LEARN_MORE_URL =
```

Comment:
Note: This URL currently 404's, I am not sure of an appropriate substitution if there is any. cc: @bschorchit